### ISSUE TITLE
[DUW] [UX] Review results page print behavior

### ISSUE THREAD

**Issue by randimays** [department-of-veterans-affairs/va.gov-cms#19758]
## Description
On the Discharge Upgrade Wizard results screen, there is a `Print this page` button at the bottom of the text content.

<img src="https://github.com/user-attachments/assets/6a4ae068-1faa-4366-9fff-90014f4a719d" width="600" />

We need to review all permutations of the DUW results screen to be sure only the desired information is presenting on the printed page. Specifically we should confirm:

- [x] How should links display? Should we see the full link URL, or hide the link?
- [x] Should we remove all buttons (e.g. "Print this page", "Back")
- [x] Is there any other page content that is not helpful in a printed context?

### User story

**AS A** Veteran who is viewing the Discharge Upgrade Wizard results page and has clicked the 'Print this page' button
**I WANT** to see only helpful, necessary information on the print view of the results page
**SO THAT** it's less visually 'busy', and also so that ink/toner isn't wasted printing works or images that aren't useful or meaningful.

### Analytics considerations
@FranECross Here's the DS event that we get when clicking the "Print this page" button. This is likely sufficient for our purposes but just wanted to drop it here for your review.
    - [Fran Cross] @randimays The DS event info we get seems to be sufficient. I see in the GA4 event: button info it provides the URL for DUW results page as well, so all the info we need is there.

<img src="https://github.com/user-attachments/assets/59000833-a179-4679-a4a1-485a35bfb769" width="600" />

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** 
- [ ] **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [x] UX provides suggested print screen alterations
- [x] UX pings @FranECross so an FE ticket can be created
- [ ] @FranECross creates FE ticket

**Comment by randimays** [comment]
@FranECross I noticed the Needs refining label was removed but we still don't have acceptance criteria for this ticket.

**Comment by FranECross** [comment]
@randimays Thanks for catching that! I'll add.

**Comment by thejordanwood** [comment]
- @FranECross I've reviewed several of the results pages and these are my answers to the questions in this ticket. 
- @randimays Please let me know if you have any other questions! Do you need a design mockup for this?

> How should links display? Should we see the full link URL, or hide the link?

The links can display the same as they do now. The link text provides context and the full URL can be used to find the right page it links to even after printing the page. If it's possible to get rid of the "(opens in a new tab)" text, then we should do that because that could reduce some of the clutter.

<img width="388" alt="Image" src="https://github.com/user-attachments/assets/00dc2453-6307-41fa-a11f-3bb3a7488054" />

> Should we remove all buttons (e.g. "Print this page", "Back")

Yes, let's remove the "Print this page" and "Back" buttons from the print screen. These aren't relevant after the page is printed. 

<img width="323" alt="Image" src="https://github.com/user-attachments/assets/05c739d0-4e18-4b08-a0e0-e64840fa15e7" />

> Is there any other page content that is not helpful in a printed context?

I think all other page content is relevant for now. I made [this comment on another ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/20272#issuecomment-2590459211), but we received feedback during the [DUW usability research](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/veteran-military-records/discharge-upgrade-wizard/research/2024-Review-Flow/research-findings.md#amount-of-content) that the amount of content on the results pages is overwhelming. I suggest we ask CAIA take a look at the content.

**Comment by FranECross** [comment]
@thejordanwood Thanks so much for review and comments! I'll ensure the Phase 2 epic has the comment from the Staging Review and will have a ticket for CAIA to review all the results page iterations to see if they can be pared down. I'll update the FE ticket with your recommendations

**Comment by thejordanwood** [comment]
Randi has said she doesn't need a design mockup for this and a FE ticket has been created. I'm going to go ahead and close this.
